webview/src/experiments/components/table/styles.module.scss

```diff
@@ -445,6 +401,11 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
     font-size: 0.8rem;
     line-height: 2rem;
     align-items: center;
+
+    &:hover {
```

Comment:
I finished most of the tasks in #2070 except for the column hover styles. Should I convert this pr to a draft until the column hover styles are added as well? 

<img width="235" alt="image" src="https://user-images.githubusercontent.com/43496356/182492516-4e29dfeb-d92d-4fe6-920e-16c0f20a826e.png">